addons/web/tests/test_login.py

```diff
@@ -18,11 +18,11 @@ def setUp(self):
         self.opener = Opener(self.env.cr)
         self.opener.cookies.set('session_id', self.session.sid, domain=HOST, path='/')
 
-    def login(self, username, password, csrf_token=None):
+    def login(self, username, password=None, csrf_token=None):
```

Comment:
This test module is explicitly testing the login page and this method is an helper for the other tests, can you restore this method signature and use `new_test_pass` in the various test cases instead?